Symfony/CS/Linter/Linter.php

```diff
@@ -106,7 +106,8 @@ private function createProcessForFile($path)
             return $this->createProcessForSource(file_get_contents($path));
         }
 
-        $process = new Process(sprintf('%s -l %s', $this->executable, ProcessUtils::escapeArgument($path)));
+        $executable = ProcessUtils::escapeArgument($this->executable).(defined('HHVM_VESION') ? '--php' : '');
```

Comment:
I doubt it would be broken on 2.x, but this stuff might change in 3.x, so that's why I don't want to depend on something I already feel is broken and I want changing.